addons/website_event/static/src/js/website_event_send_email.js

```diff
@@ -0,0 +1,94 @@
+    selector: '.o_wevent_send_by_email_widget',
+    events: {
+        'click .o_wevent_js_send_email': '_onSendEmailClick',
+        'click .o_wevent_send_by_email_info .btn-close': '_onCloseInfoClick',
+    },
+
+    init() {
+        this.notification = this.bindService("notification");
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onSendEmailClick: async function (ev) {
+        ev.target.getElementsByTagName('i')[0].classList.remove('d-none');
+        ev.target.setAttribute('disabled', '');
+
+        const attendeesIds = JSON.parse(ev.target.dataset.attendeesIds) || [];
+        const eventId = parseInt(ev.target.dataset.eventId);
+        const ticketsHash = ev.target.dataset.ticketsHash;
+        const emails = document.querySelector('#o_wevent_send_by_email_input').value;
+        const emailInfo = checkEmailValidity(', ', emails)
+
+        if (emails === "" || emailInfo.invalidEmails.length) {
+            this._showError();
+        } else {
+            this._hideError();
+            await rpc(`/event/${ eventId }/my_tickets_by_email`, {
+                'emails': emails,
+                'registration_ids': attendeesIds,
+                'tickets_hash': ticketsHash,
+            });
+            this._showInfoMsg();
+        }
+        ev.target.removeAttribute('disabled');
+        ev.target.getElementsByTagName('i')[0].classList.add('d-none');
+    },
+
+    /**
+     * @private
+     * Show error message if email is invalid by adding the bootstrap class
+     * 'is-invalid' on the input field and its parent. The parent is the div that
+     * contains the input field and the error message and has the class 'has-validation'.
+     * For more information : https://getbootstrap.com/docs/5.3/forms/validation/#server-side
+     */
+    _showError: function () {
+        const inputElement = document.querySelector('#o_wevent_send_by_email_input');
+        inputElement.classList.add('is-invalid');
+        inputElement.parentElement.classList.add('is-invalid');
+    },
+
+    /**
+     * @private
+     * Hide error message if email is valid by removing the bootstrap class
+     * 'is-invalid' on the input field and its parent. The parent is the div that
+     * contains the input field and the error message and has the class 'has-validation'.
+     * For more information : https://getbootstrap.com/docs/5.3/forms/validation/#server-side
+     */
+    _hideError: function () {
+        const inputElement = document.querySelector('#o_wevent_send_by_email_input');
+        inputElement.classList.remove('is-invalid');
+        inputElement.parentElement.classList.remove('is-invalid');
+    },
+
+    _showInfoMsg: function () {
+        const inputMsgDiv = document.querySelector('.o_wevent_send_by_email_widget div');
```

Comment:
Please add a selector instead of using Div. Two reasons for this: we tend to avoid using the dom structure to name things as, well, it ends up changing at some point ^^. Also, it makes it easier to read and to find.
So feel free to add a selector like `o_wevent_send_by_email_input_container` or something along those lines.